### PR TITLE
Improve group and release-notes commands descriptions

### PR DESCRIPTION
**Pull request by Jamminroot**
As per #1213, this PR is to improve help output of commands

### CODE REVIEW

File: src/commands/distribute/release.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default class ReleaseBinaryCommand extends AppCommand {
   @hasArg
   public storeName: string;
 
-  @help("Release notes text")
+  @help("Release notes text (markdown supported, 5000 characters max)")
```
Comment (DmitriyKirakosyan) [inline src/commands/distribute/release.ts:72]: Markdown is not supported for this option, actually. See https://github.com/microsoft/appcenter-cli/issues/1184#issuecomment-789300556

Comment (DmitriyKirakosyan) [reply src/commands/distribute/release.ts:72]: Add his note about 5000 characters to release-notes-file, and for that option you can specify that markdown is supported.

Comment (Jamminroot) [reply src/commands/distribute/release.ts:72]:
```suggestion
  @help("Release notes text")
```

**Review by MatkovIvan — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

**Comment by DmitriyKirakosyan** [comment]
LGTM

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**